library/std/src/sync/mpsc/mod.rs

```diff
@@ -315,6 +315,8 @@ mod cache_aligned;
 /// The receiving half of Rust's [`channel`] (or [`sync_channel`]) type.
 /// This half can only be owned by one thread.
 ///
+/// Receiver can only be sent between threads if `T` is [`Send`][`core::marker::Send`]
```

Comment:
You don't need the link target if the link text already resolves to the right thing:
```suggestion
/// A receiver can only be sent between threads if `T` is [`Send`].
```
(Same below.)